src/main/resources/public/js/app.js

```diff
@@ -82,15 +84,34 @@ for (var i = 0; i < gameModel.computerMisses.length; i++) {
 for (var i = 0; i < gameModel.computerHits.length; i++) {
    $( '#TheirBoard #' + gameModel.computerHits[i].Down + '_' + gameModel.computerHits[i].Across ).css("background-color", "red");
 }
-
 for (var i = 0; i < gameModel.playerMisses.length; i++) {
    $( '#MyBoard #' + gameModel.playerMisses[i].Down + '_' + gameModel.playerMisses[i].Across ).css("background-color", "green");
 }
 for (var i = 0; i < gameModel.playerHits.length; i++) {
    $( '#MyBoard #' + gameModel.playerHits[i].Down + '_' + gameModel.playerHits[i].Across ).css("background-color", "red");
 }
 
+/* Waits a moment for the browser to finish updating, then checks to see
+ * if a "Game Over" alert has already been thrown. If not, it alerts the
+ * the user with "Game Over" and sets "Alerted" to true.
+ */
+setTimeout( function(){
```

Comment:
If you place it inside request.done, it will guarantee that this happens after the board is updated (because request.done doesn't get called until the response is received).